integrations/atlassian/jira/webhooks.go

```diff
@@ -1,19 +1,17 @@
 package jira
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
 	"fmt"
-	"io"
-	"net/http"
 	"os"
 	"slices"
 	"strings"
 	"time"
 
 	"go.uber.org/zap"
 
+	"go.autokitteh.dev/autokitteh/integrations/common"
```

Comment:
In general I don't like "common" packages

Reply:
I agree, it's actually noted as a bad practice in the Go style guide, but I don't have a better name yet.